test/errorOutput.bal

```diff
@@ -0,0 +1,112 @@
+import ballerina/io;
+import ballerina/file;
+import ballerina/regex;
+
+public function main() {
```

Comment:
If you are going to do this in Ballerina, wouldn't it be better to add an option to `main` to print out the HTML directly.

Reply:
Will that be useful to the user given they already get the error message and all this is doing is aggregating error messages (which is useful to us because that helps with figuring out if our error messages are consistent and highlight the correct part of the source).

Reply:
It doesn't have to be useful for the end-user: the option would be mainly for our internal use (like `--showTypes`). Plus we are the only users for the moment.  The question is what's convenient for us. It think it's easier if we can just do `java --htmlErrors nballerina.jar testSuite/*/*-e.bal` and get the HTML on stdout.